rust/syntect-plugin/src/main.rs

```diff
@@ -50,6 +50,7 @@ type EditBuilder = DeltaBuilder<RopeInfo>;
 pub enum EditType {
     Insert,
     Newline,
+    Other,
```

Comment:
really the longer term solution here is to actually expose and use the `EditType` defined in core and use serde